### PR TITLE
added method descriptions, removed tests, cleaned up docs

### PR DESCRIPTION
**Pull request by joage**
- parameter descriptions
- return values

### CODE REVIEW

File: Scripts/Embrace.cs [hunk 1]
```diff
@@ -228,34 +315,60 @@ public void LogMessage(string message, EMBSeverity severity, Dictionary<string,
...
+        /// Get the user identifier assigned to the device by Embrace.
+        /// </summary>
+        /// <returns>the device identifier created by Embrace</returns>
         public string GetDeviceId()
         {
             return provider.GetDeviceId();
         }
 
+        /// <summary>
+        /// Starts a view. There is a limit to 10 "started" views.
```
Comment (joage) [inline Scripts/Embrace.cs]: @elanz-embrace Please verify this, need to add anything?

File: Scripts/Embrace.cs [hunk 2]
```diff
@@ -228,34 +315,60 @@ public void LogMessage(string message, EMBSeverity severity, Dictionary<string,
...
+        /// <param name="name">name of the view state as it will show up on our dashboard</param>
+        /// <returns>a boolean indicating whether the operation was successful or not</returns>
         public bool StartView(string name)
         {
             if (name == null) { NoNullsError(); return false; }
             return provider.StartView(name);
         }
 
+        /// <summary>
+        /// Closes the view state for the specified view or logs a warning if the view is not found.
```
Comment (joage) [inline Scripts/Embrace.cs:359]: @elanz-embrace same here

File: Scripts/Embrace.cs [hunk 3]
```diff
@@ -96,12 +100,21 @@ private void NoNullsError()
             Debug.LogError("null is not allowed through the Embrace SDK.");
         }
 
+        /// <summary>
+        /// Starts instrumentation of the Android application using the Embrace SDK. This should be called during creation of the application, as early as possible.
```
Comment (elanz-embrace) [inline Scripts/Embrace.cs]: Should be generic, this layer isn't specific to android.

File: Scripts/Embrace.cs (hunk 1 above)
Comment (elanz-embrace) [reply Scripts/Embrace.cs]: we might use the word "open" instead of "started" for this, otherwise this feels correct.

**Review by elanz-embrace — APPROVED** [review]
I think just that one android issue should be fixed.

File: Scripts/Embrace.cs (hunk 1 above)
Comment (joage) [reply Scripts/Embrace.cs]: Changed to "open"

File: Scripts/Embrace.cs (hunk 3 above)
Comment (joage) [reply Scripts/Embrace.cs]: removed "android"

**Pull request merged**